src/brb_membership.rs

```diff
@@ -44,10 +72,16 @@ impl<A: Ord> Reconfig<A> {
     }
 }
 
+/// Enumerates the different Ballot types that can be voted on
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 pub enum Ballot<A: Ord, S: Ord> {
+    /// A proposal to add or remove an actor to the voting group
     Propose(Reconfig<A>),
+
+    /// merge split votes
     Merge(BTreeSet<Vote<A, S>>),
+
+    /// Supermajority achieved over split vote
```

Comment:
```suggestion
    /// Supermajority achieved over a set of `Reconfig`s
```

Reply:
kk

Reply:
@dan-da expanding a bit, we don't always have split votes (ie. if we have just one reconfig proposed, then we won't have a split)